benchmark_runner/workloads/bootstorm_vm.py

```diff
@@ -92,7 +106,7 @@ def _get_bootstorm_vm_elapsed_time(self, vm_name: str, vm_node: str) -> dict:
         """
         if vm_node:
             delta = round((time.time() - self._bootstorm_start_time[vm_name]) * self.MILLISECONDS, 3)
-            data = {'vm_name': vm_name, 'node': vm_node, 'bootstorm_time': delta, 'vm_ssh': int(bool(vm_node)),}
+            data = {'vm_name': vm_name, 'node': vm_node, 'bootstorm_time': delta, 'virtctl_vm': int(bool(vm_node)),}
```

Comment:
Rather than name this "vm_ssh" or "virtctl_vm", name it something like "vm_status" to reflect its function.

Reply:
OK.